Update_user.css

```diff
@@ -61,7 +100,7 @@ nav a{
 
 
 .img_2{
-	background-color: green;
+	background-color: white;
```

Comment:
I make the background of the image to white because if they images can't load up then the background would look nicer if it was white